metadata_service/proxy/base_proxy.py

```diff
@@ -151,3 +151,42 @@ def get_resources_using_table(self, *,
                                   id: str,
                                   resource_type: ResourceType) -> Dict[str, List[DashboardSummary]]:
         pass
+
+    # NB: the following methods exist solely for testing in the roundtrip tests.
```

Comment:
does all the put methods are solely used for round trip testing?

Reply:
Yep, exactly!